metacontroller_pilot/metacontroller_pilot/metacontroller_sim.py

```diff
@@ -47,66 +29,45 @@
 from rqt_gui_py.plugin import Plugin
 from std_msgs.msg import Float32, Header
 from system_modes.srv import ChangeMode
-
+from rcl_interfaces.msg import Log
 
 class Metacontroller(Node):
-    def __init__(self, node_name, mode_name):
+    def __init__(self):
         super().__init__('metacontroller')
+        self.rosout_sub_ = self.create_subscription(
+          Log,
+          "/rosout",
+          self.rosout_cb, 1)
+        self.current_mode = 'NORMAL'
+    def change_mode(self, node_name, mode_name):
         cli = self.create_client(ChangeMode, '/'+node_name+'/change_mode')
         while not cli.wait_for_service(timeout_sec=1.0):
             print('service not available, waiting again...')
         req = ChangeMode.Request()
+        req.node_name = node_name
```

Comment:
This causes an error with newer versions of [`system_modes`](https://github.com/micro-ROS/system_modes/commit/b6d35bce3ce9a2094c191779db201c7e6be0b820):
```console
ros2 run metacontroller_pilot metacontroller
Traceback (most recent call last):
  File "/home/parallels/pilot_urjc_ws/install/metacontroller_pilot/lib/metacontroller_pilot/metacontroller", line 11, in <module>
    load_entry_point('metacontroller-pilot', 'console_scripts', 'metacontroller')()
  File "/home/parallels/pilot_urjc_ws/build/metacontroller_pilot/metacontroller_pilot/metacontroller_sim.py", line 67, in main
    node.change_mode("pilot", '__DEFAULT__')
  File "/home/parallels/pilot_urjc_ws/build/metacontroller_pilot/metacontroller_pilot/metacontroller_sim.py", line 47, in change_mode
    req.node_name = node_name
AttributeError: 'ChangeMode_Request' object has no attribute 'node_name'
```
See https://github.com/micro-ROS/system_modes/issues/24

```suggestion

```

Reply:
I am using the `feature/rules` branch, and this field it is necessary. I have changed the dependencies.repos file to download this branch instead of master